FlowCrypt/Functionality/DataManager/DataService.swift

```diff
@@ -111,7 +111,7 @@ extension DataService: DataServiceType {
 
     func validAccounts() -> [User] {
```

Comment:
`validAccounts` sould be called `getFinishedSetupUsersExceptCurrentUser` based on what it does now. Naming the methods by what they actually do sometimes highlights the ridiculousness of their logic.

More properly speaking, it should be renamed as `getFinishedSetupUsers` and it should not filter by `currentEmail`. The callers of that method should do the filtering, if that's what they need. If this method is used in many places that all need the filtering, then the method should be changed to `getFinishedSetupUsers(exceptUserEmail: String)` for clarity.

Reply:
We use this in one place - app menu, when we show list of possible accounts to switch to.